app/models/user.rb

```diff
@@ -29,6 +29,11 @@ def email_required?
     !(Portus::LDAP.enabled? && !ldap_name.nil?)
   end
 
+  # Returns true if this user has an email properly set.
+  def email?
```

Comment:
not needed with activemodel/activerecord. see https://github.com/rails/rails/blob/v3.2.3/activerecord/lib/active_record/attribute_methods/query.rb